modules/ROOT/pages/addition/agent-installation/manual.adoc

```diff
@@ -175,7 +298,7 @@ The rest of NOM still works.
 | 9500
 
 | `CONFIG_INSTANCE_n_LOG_CONFIG_PATH`
-| Path to the instance <<server_log_config,log4j config file>>.
+| Path to the instance xref:querylog[log4j config file].
```

Comment:
I missed this completely 🤦. 

Do you think it would be even better if we re-instated a separate section for the location of that file? Maybe as a subsection of the querylog section?

Reply:
Yes, also to reduce duplication, we should do something here.

I suggest putting the sentence " If CONFIG_INSTANCE_n_QUERY_LOG_PORT is not set, the feature is treated as disabled. The rest of NOM still works." to the description of CONFIG_INSTANCE_n_QUERY_LOG_PORT.

Then move "The location of server log configuration ..." including the IMPORTANT section to "Query log collection" on the page "Neo4j instance requirements" including adjusting the links from CONFIG_INSTANCE_n_LOG_CONFIG_PATH to point there.

WDYT?

Reply:
That sounds like a good plan 👍

Reply:
OK, so I'll commit it to your branch so that you can review.